libs/core/langchain_core/output_parsers/openai_tools.py

```diff
@@ -44,7 +44,7 @@ def parse_tool_call(
         "args": function_args or {},
     }
     if return_id:
-        parsed["id"] = raw_tool_call["id"]
+        parsed["id"] = raw_tool_call.get("id")
```

Comment:
this is breaking if one expects a KeyError when using `JsonOutputToolsParser(return_id=True)` and there are no IDs.